fastlane_bot/events/managers/base.py

```diff
@@ -238,6 +251,11 @@ def exchange_name_from_event(self, event: Dict[str, Any]) -> str:
         str
             The exchange name.
         """
+        if 'id' in event['args']:
+            carbon_controller_address = event['address']
+            for ex in self.cfg.CARBON_CONTROLLER_MAPPING:
```

Comment:
Semantics: probably better using `exchange_name` to keep it consistent with the rest of this function.
And it's probably true for a bunch of other places below and above (wherever you're returning the name of the exchange and not the exchange object).

Reply:
due to the number of related changes involved with handling every similar occurrence, I have created a new issue for this https://github.com/bancorprotocol/fastlane-bot/issues/469